cmd/agent-mobile/pkg/wrappers/rest/didclient.go

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package rest // nolint: dupl
+
+import (
+	"github.com/trustbloc/agent-sdk/cmd/agent-mobile/pkg/wrappers/models"
+	"github.com/trustbloc/agent-sdk/pkg/controller/command/didclient"
+)
+
+// DIDClient contains necessary fields to support its operations.
+type DIDClient struct {
```

Comment:
Do we have TODOs for the rest of the lifecycle?

Create, Update, Recover, Deactivate?